README.md

```diff
@@ -172,6 +172,10 @@ Using external grep-like program to search `display` and replace it to `show`, b
             any floating attributes.]],
         default = true
     },
+    enable_mouse = {
```

Comment:
I think `enable_mouse_remap` is better, `enable_mouse` makes users think bqf will change their `mouse` option.